test/boost/cql_functions_test.cc

```diff
@@ -66,6 +66,7 @@ SEASTAR_TEST_CASE(test_functions) {
                     }
                 }
                 virtual void visit(const result_message::bounce_to_shard& rows) override { throw "bad"; }
+                virtual void visit(const result_message::exception&) override { throw "bad"; }
```

Comment:
Changelog calls it result_message::error, needs adjusting.

Reply:
Done